src/GovUk.Education.ExploreEducationStatistics.Common/Model/DataFileInfo.cs

```diff
@@ -1,7 +1,6 @@
 using System;
-using GovUk.Education.ExploreEducationStatistics.Common.Model;
 
-namespace GovUk.Education.ExploreEducationStatistics.Admin.Models
+namespace GovUk.Education.ExploreEducationStatistics.Common.Model
```

Comment:
I'm not sure why this was necessary?

Reply:
Nope, think I must have moved it earlier on before I realised a better way of doing things was using `BlobInfo`.